src/api/routes.ts

```diff
@@ -106,6 +106,10 @@ export function publishPackage(req: auth.AuthRequest, res: express.Response): vo
     });
 }
 
+export function whoami(req: auth.AuthRequest, res: express.Response): express.Response {
+  return res.status(200).json({ "username": res.locals.remote_user.name });
```

Comment:
how could this work? where do we store `remote_user.name` in `res.locals`? not sure but I believe you need to work here with tokens...